File: Arnav Sharma/Assignment-0/if-else/greatest.java
// Take two int values from user and print greatest among them.

import java.util.Scanner;

public class greatest{
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter the first number : ");
        int first = sc.nextInt();
        System.out.print("Enter the second number : ");
        int second = sc.nextInt();
        if(first > second)
            System.out.println(first+" is greater than "+second);
        else if(second > first)
            System.out.println(second+" is greater than "+first);
        else
            System.out.println("Both numbers are Equal");
        sc.close();
    }
}

File: Sawtantar/if else/greatest.cpp
#include <iostream>

using namespace std;

int
main ()
{
  int a, b;
  cout << "Enter Value of Number A " << endl;
  cin >> a;
  cout << endl << "Enter Vaue of  Number B " << endl;
  cin >> b;
  if (a > b)
    {
      cout << endl << "Number A is greater than B";
    }
  else
    {
      cout << endl << "Number B is greater than A";
    }
    return 0;
}
